src/blocks/Banner/schema.ts

```diff
@@ -14,9 +14,11 @@ export const BannerCardProps = {
         ...AnimatableProps,
         title: {
             type: 'string',
+            content: 'text',
         },
         subtitle: {
             type: 'string',
+            content: 'yfm',
```

Comment:
is it standard json schema field or custom?
what is the meaning of the field does it contain content type or what?